index.html

```diff
@@ -47,6 +47,183 @@ <h1>THIERRY <br> NOUMBUH</h1>
             <img src="assets/" alt="">
           </div>
         </section>
+
+        <section class="project-section-container">
+            <div class="project-section">
+                <h2>projects</h2>
```

Comment:
Keep doing great work @axcell-0 👍, but let's make some changes to stick to the design 💪   

- [ ] Kindly make sure the `languages section` look like the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃. Please, fix the `margin`, `font`, and `padding` 💪 .

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-06-01 155956](https://github.com/axcell-0/Thierrys-portfolio/assets/57408419/7f132e51-c7c6-48d2-8ad4-7be3b848618b)| |![Screenshot 2023-06-01 160149](https://github.com/axcell-0/Thierrys-portfolio/assets/57408419/e372c3a9-05bb-4454-9e11-e9d1997b5305)| |